release_notes.md

```diff
@@ -4,6 +4,35 @@ title: Release Notes
 permalink: /release_notes/
 ---
 
+## 1.9.0.0.7062 - October 14th 2017
+
+* Add option to securely save lobby credentials (#2395).
+* Fix console opening when playing a network game (#2289).
+* Fix threading errors reported when starting Map Creator (#2238).
+* Improve security of network game authentication (#2107).
+* Add option to securely save PBEM/PBF credentials (#1955).
+* Fix out-of-date map check (#1695).
+* Fight a single battle automatically
+* Upgrade retreat options - no double prompt with one valid territory
+  * don't offer prompt when only defenseless units remain
+* Show dice details by default in PBF/PBEM
+* Auto fight battles with no defenders who can fire and no dependent battles
+* Add experimental savegame format (#2128)
+* Setting Menu Overhaul (#2137)
+* Bugfix: Maps can have AI-only players(#2149)
+* Removal of the Beta Map Creator (#2156)
+* Enabled TripleA to download Maps via a browser (#2274)
+* Updated many outdated links
+* Improved internal password security (#2101)
+* General smaller performance improvements
+* Several smaller usability improvements
+* Using Java 1.8.0_144 for the installer now (#2384)
+* Removed the 'old jar' functionality (#2284)
+
+And many more internal changes, bugfixes etc.
+See the full List [here](https://github.com/triplea-game/triplea/pulls?utf8=%E2%9C%93&q=merged%3A%3E2017-07-22)
```

Comment:
@ssoloff btw. I used this link to get all PRs being merged since the last entry in this file, I didn't paricularily checked who created Pr xyz, I just looked at the title and copied the PR number into the changelog

Reply:
Thanks, @RoiEXLab.  And I agree about having some kind of consistent naming convention to differentiate features, bug fixes, and "everything else".  As well as choosing a consistent sort order. :smile:

Reply:
Another possibility is to use something like the format suggested by [keep a changelog](http://keepachangelog.com).  That is, instead of identifying items as _features_, _bug fixes_, etc., we group them by _added_, _changed_, and _removed_.